File: day_five/countdown.js
let num = prompt("Enter a number: ");
num = Number(num);
let header = document.querySelector("h1");

while (num > 0) {
    alert("Going in " + num);
    num -= 1;
}
if (num == 0) {
    header.innerHTML = "Explode!";
}

File: day_five/countdown.cpp
#include <iostream>

int main() {

    int number;

    std::cout << "Enter a number: ";
    std::cin >> number;

    while (number > 0) {
        std::cout << "Going in " << number << '\n';
        number--;
    }

    if (number == 0) {
        std::cout << "Explode!\n";
    }

    return 0;
}